### ISSUE TITLE
Excellent Detection Performance but not very good in classification

### ISSUE THREAD

**Issue by duchieuphan2k1** [tinyvision/DAMO-YOLO#96]
### Before Asking

- [X] I have read the [README](https://github.com/tinyvision/DAMO-YOLO/blob/master/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [X] I want to train my custom dataset, and I have read the [tutorials for finetune on your data](https://github.com/tinyvision/DAMO-YOLO/blob/master/README.md#Finetune%20on%20your%20data) carefully and organize my dataset correctly; 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。

- [X] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the DAMO-YOLO [issues](https://github.com/tinyvision/DAMO-YOLO/issues) and found no similar questions.

### Question

I have trained damo-yolo-m on my data (2 classes). The model can detect object very well, with no missing, no excessive. But the classification is not very good. So is there any idea or configuration for me to improve the classification accuracy or do some tradeoff between detection and classification tasks? 

### Additional

Thanks in advance!

**Comment by cwhgn** [comment]
Thanks for your interests on DAMO-YOLO. You can provide a higher weight for the classification loss, or train more epochs to see whether the classification loss would be further reduced. Besides, it's better to check your classification labels to see whether there are noise labels and category imbalance problem.
